doc/DataPlaneCodeGeneration/DeveloperDrivenEvolution.md

```diff
@@ -122,6 +146,9 @@ namespace Azure.AI.MetricsAdvisor
         // Suggest appending value to method name
         public virtual async Task<Response<MetricFeedback>> GetMetricFeedbackValueAsync(Guid feedbackId, CancellationToken cancellationToken = default)
         {
+            // Open a new scope here and the inner scope will be suppressed
```

Comment:
It would be good to mention in the comment that we open the scope in the convenience method because it has a different name.